hw/formal/formal_conn

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
```

Comment:
we usually name bash script with `.sh`

Reply:
Thanks Weicai for the reminder, let me update the naming :)